test/unit/components/patientcard.test.js

```diff
@@ -26,6 +40,16 @@ describe('PatientCard', function () {
       expect(elem).to.be.ok;
       expect(console.error.callCount).to.equal(0);
     });
+
+    it('should render upload button if user has permissions set', function() {
```

Comment:
For these, the descriptions should be something like:
'should render upload button if user has upload permission'